client/my-sites/purchases/main.tsx

```diff
@@ -60,3 +61,34 @@ export function PurchaseDetails( {
 		</Main>
 	);
 }
+
+export function PurchaseCancel( {
+	purchaseId,
+	siteSlug,
+}: {
+	purchaseId: number;
+	siteSlug: string;
+} ) {
+	const translate = useTranslate();
+
+	const getManagePurchaseUrlFor = ( targetSiteSlug: string, targetPurchaseId: string | number ) =>
+		`/purchases/${ targetSiteSlug }/${ targetPurchaseId }`;
+
+	return (
+		<Main className="purchases is-wide-layout">
+			<DocumentHead title={ translate( 'Cancel purchase' ) } />
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **34** times:
`translate( 'Cancel Purchase' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-level-billing-cancel-purchase-page)

Reply:
They are the same. This is just sentence-casing the title.